enhancements/etcd/automated-backups-no-config.md

```diff
@@ -0,0 +1,256 @@
+* Cons
+  - CSI is not an option for OCP `BM` or `SNO`.
+  - Some cloud providers allocate storage to be accessible only within the same availability zone. Since Openshift cluster allocate each master node on a different zone for high availability purposes, the backup will not be accessible if the master node where the backup was taken is no longer accessible.
+
+### Local Storage
+
+Relying on local storage works for all Openshift variants (i.e. cloud based, SNO and BM). There are three possible approaches using local storage, each is detailed below.
+
+#### Local Storage Operator
+
+Utilising `local storage operator` is a proper solution for all Openshift variants. However, installing a whole operator is too much overhead.
+
+#### Local volumes
+
+A `localVolume` mounts a path from the node's file system tree or a separate disk as a volume into a Pod.
+
+Utilising a `local StorageClass` with no provisioner, guarantees a behaviour similar to dynamic provisioning.
+Upon creating a `PVC` with the `local StorageClass`, a `PV` will be allocated using `localVolume` and bound to the `PVC`.
+Since the `PV` is actually a path on the node's file system, the etcd backup will be always available regardless of the Pod's status, as long as the node itself is still accessible.
+using `localVolume` and the node affinity within the `PV` manifest forces the backup pod to be scheduled on a specific node, where the volume is attached.
+Had the Pod been replaced, the new Pod is guaranteed to be scheduled on the same node, where the backup exists.
+This deterministic scheduling behaviour is a property of `localVolume`, which is not the case with `hostPath` volumes.
+`localVolume` is handled by the PV controller and the scheduler in different manner, in fact it was created to resolve issues with `hostPath`
+
+* Pros
+  - It supports all Openshift variants, including `SNO` and `BM`.
+  - The Backup Pod will be scheduled always to this master node where the backup data exists.
+  - Using a separate disk for backups (e.g. SSD) is possible, since local volume allows mounting it into a Pod.
+  - Having the deterministic Pod scheduling is vital, since the restoration needs the recent backup in place to work.
+* Cons
+  - If the master node where the `PV` is mounted became unhealthy/unavailable/unreachable, then the backups are no longer accessible, also taking new backups is no longer possible.
+
+#### Host Path volumes
+
+A `hostPath` volume mounts a path from the node's file system as a volume into a Pod.
+
+* Pros
+  - It supports all Openshift variants, including `SNO` and `BM`.
+* Cons
+  - `hostPath` could have security impact as it exposes the node's filesystem.
+  - No scheduling guarantees for the pod using `hostpath` as is the case with `localVolume`. The pod could be scheduled on a different node from where the hostPath volume exist.
```

Comment:
If you have a particular want, you could schedule the pods using the CEO to guarantee where they end up